src/script/ui/ContextMenu.tsx

```diff
@@ -52,21 +53,31 @@ const cleanUp = () => {
 };
 
 const ContextMenu: React.FC<ContextMenuProps> = ({entries, defaultIdentifier = 'ctx-menu-item', posX, posY}) => {
-  const [mainDiv, setMainDiv] = useState<HTMLDivElement>();
+  const [mainElement, setMainElement] = useState<HTMLUListElement>();
   const [selected, setSelected] = useState<ContextMenuEntry>();
 
   const style = useMemo<React.CSSProperties>(() => {
-    const left = mainDiv && window.innerWidth - posX < mainDiv.offsetWidth ? posX - mainDiv.offsetWidth : posX;
+    const left =
+      mainElement && window.innerWidth - posX < mainElement.offsetWidth ? posX - mainElement.offsetWidth : posX;
     const top = Math.max(
-      mainDiv && window.innerHeight - posY < mainDiv.offsetHeight ? posY - mainDiv.offsetHeight : posY,
+      mainElement && window.innerHeight - posY < mainElement.offsetHeight ? posY - mainElement.offsetHeight : posY,
       0,
     );
     return {
       left,
       top,
-      visibility: mainDiv ? 'unset' : 'hidden',
+      visibility: mainElement ? 'unset' : 'hidden',
     };
-  }, [mainDiv]);
+  }, [mainElement]);
+
+  useEffect(() => {
+    if (selected) {
+      const selectedButton = document.querySelector(
+        `#${selected.label.split(' ').join('_').toLowerCase()}-button`,
+      ) as HTMLButtonElement;
+      selectedButton.focus();
```

Comment:
Just to be on the safe side
```suggestion
      selectedButton?.focus();
```

Reply:
you are right - fixed, thanks